src/main/java/com/staff_management_system/com/database_connection_package/DatabaseConnect.java

```diff
@@ -0,0 +1,51 @@
+package com.database_connection_package;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnect {
+    /**
+     * Connection object to open port to db
+     */
+    private final ThreadLocal<Connection> aConnection = new ThreadLocal<>();

```

Comment:
why threadlocal?

Reply:
The threadlocal was suggested by IntelliJ as a solution to one of the warnings.
I will work around it to fix it.